content/spin/sqlite-api-guide.md

```diff
@@ -0,0 +1,150 @@
+title = "SQLite Storage"
+template = "spin_main"
+date = "2023-04-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/sqlite-api-guide.md"
+
+---
+- [Using SQLite Storage From Applications](#using-sqlite-storage-from-applications)
+- [Preparing a SQLite Database](#preparing-a-sqlite-database)
+- [Custom SQLite Databases](#custom-sqlite-databases)
+- [Granting SQLite Database Permissions to Components](#granting-sqlite-database-permissions-to-components)
+
+Spin provides an interface for you to persist data in a SQLite database managed by Spin. This database allows Spin developers to persist relational data across application invocations.
+
+{{ details "Why do I need a Spin interface? Why can't I just use my own external database?" "You can absolutely still use your own external database either with the MySQL or Postgres APIs. However, if you're interested in quick, local relational storage without any infrastructure set-up then Spin's SQLite database is a great option." }}
+
+## Using SQLite Storage From Applications
+
+> The SQLite API is experimental and likely to change. Do not depend on the current API for production applications.
```

Comment:
Do we have any other more prominent display of the experimental nature of this feature?